pallenec

```diff
@@ -21,6 +22,7 @@ p:mutex(
     p:flag("--emit-lua",  "Generate a .lua file instead of an .so file"),
     p:flag("--compile-c", "Compile a .c file generated by --emit-c"),
     p:flag("--print-ir",  "Print a human-readable version of the program's IR"),
+    p:flag("--tokens", "Print the tokens recognized by the lexer"),
```

Comment:
This is a nice feature! But could we change it into `--dump=lexer`, to be more consistent  with the others?